iface-ng/README.md

```diff
@@ -0,0 +1,25 @@
+# iface-ng
+
+Display information about network interfaces, more flexible than default `iface`.
+
+``` ini
+[iface-ng]
+command=/home/roobre/Devel/i3blocks-contrib/$BLOCK_NAME/$BLOCK_NAME -psd ''
+label=eth0:
+instance=eth0
+interval=10
+```
+
+## Usage
+
+    Usage: $0 [-d down] [-n "no ip"] [-c FF0000] [-C 00FF00] [-e] [-p] [-s] [-6]
+        -d down: Placeholder text if the specified interface is down. If set to an empty string,
+           the entire block is hidden (useful to save space on small screens).
+        -n "no ip": Placeholder text to display when the interface is up but it does not
+           have any ip address yet.
+        -c FF0000: Text color when interface is down.
```

Comment:
[Here](https://github.com/vivien/i3blocks-contrib/pull/23#discussion_r48615703) you told me to use ./iface-ng (?)